aperture.go

```diff
@@ -163,6 +164,7 @@ type Aperture struct {
 
 	etcdClient    *clientv3.Client
 	db            *sql.DB
+	lncNodeConn   *lnc.NodeConn
```

Comment:
maybe makes sense to put inside the challenger instead?

Perhaps we make the challenger an interface and then let the LNC conn be one implementation of the Challenger? checkout what I did here for adding the LNURL challenger: https://github.com/lightninglabs/aperture/pull/77/commits/0c780a819b2b9d088c91f1989ec55babbc70f5da

Reply:
+1 -- then also able to add other versions outside the package as well

Reply:
Much better your way, thanks @ellemouton :tada: